SDL/Event.cuh

```diff
@@ -14,13 +14,27 @@
 
 #include "allocate.h"
 
+// Temporary alpaka statements
+using Dim = alpaka::DimInt<3u>;
+using Idx = std::size_t;
+using Vec = alpaka::Vec<Dim,Idx>;
+using QueueProperty = alpaka::NonBlocking;
+using WorkDiv = alpaka::WorkDivMembers<Dim, Idx>;
+
 namespace SDL
 {
     class Event
     {
     private:
         cudaStream_t stream;
         bool addObjects;
+
+#ifdef ALPAKA_ACC_GPU_CUDA_ENABLED
```

Comment:
Out of curiosity, is this needed in our case? Aren't we always compiling for the cuda backend? Or is it a guard for the future (that we didn't have before, as we were defining the accelerator without it previously)?

Reply:
It's actually needed because of the LST.cc that's present in the SDL directory now. The LST.cc is compiled with the host compiler of nvcc, and it throws an error without the ifdef here.